discovery-provider/src/tasks/playlists.py

```diff
@@ -64,10 +64,10 @@ def playlist_state_update(
 
             num_total_changes += processedEntries
 
-    logger.info(f"[playlist indexing] There are {num_total_changes} events processed.")
+    logger.info(f"index.py | playlists.py | There are {num_total_changes} events processed.")
```

Comment:
if this is to make it easier to filter i'm not sure it's the best way - seems like a log filtering optimization we are doing in the code itself

Reply:
like if the goal here was just to filter by `index.py` i understand, but this is literally not accurate - `index.py | playlists.py | ....` representing this task is called from this file seems very odd to me. don't feel too strongly if you want to move ahead with this, just seems weird and probably solved with a grep optimization